frontend/add-leads/add-edit-materialEntry.css

```diff
@@ -0,0 +1,20 @@
+.container {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.deleteButton {
+  color: red;
+  cursor: pointer;
```

Comment:
Our buttons don't have `pointer` for the cursor - let's delete this to be consistent:

```suggestion
```